src/ol_concourse/pipelines/infrastructure/keycloak/docker_packer_pulumi_pipeline.py

```diff
@@ -86,7 +86,10 @@ def build_keycloak_pipeline() -> Pipeline:
     # Repo: https://github.com/daniel-frak/keycloak-user-migration
     # Use: Migrating users from open discussions
     user_migration_spi = github_release(
-        Identifier("user-migration-spi"), "daniel-frak", "keycloak-user-migration"
+        Identifier("user-migration-spi"),
+        "daniel-frak",
+        "keycloak-user-migration",
+        KEYCLOAK_VERSION,
```

Comment:
We should set the version to be `22.0` so that patch releases will get pulled in.

Reply:
If we set it to that, wouldn't this break as it would be unable to find the keycloak version to deploy? https://github.com/mitodl/ol-infrastructure/blob/main/src/bilder/images/keycloak/deploy.py#L75

Reply:
No, because that is getting set based on the value that we are setting on line 185.